functions/src/firestore-collections.ts

```diff
@@ -1,7 +1,8 @@
-import type { Lobby, PrivatePlayer, PromptAnswer } from "./firestore-types/lobby";
+import { ChatMessage, ChatRoom, Lobby, PrivatePlayer, PromptAnswer } from "./firestore-types/lobby";
+import { db } from "./app";
+import { firestore } from "firebase-admin";
```

Comment:
let's use the import from "firebase-admin/firestore" instead of the firestore namespace for types